protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -554,6 +577,26 @@ func renderMessagesAsDefinition(messages messageMap, d openapiDefinitionsObject,
 	return nil
 }
 
+func checkVisibility(r *visibility.VisibilityRule, reg *descriptor.Registry) bool {
+	if r == nil {
+		return true
+	}
+
+	restrictions := strings.Split(strings.TrimSpace(r.Restriction), ",")
+	// No restrictions results in the element always being visible
+	if len(restrictions) == 0 {
+		return true
+	}
+
+	for _, restriction := range restrictions {
+		if reg.GetVisibilityRestrictionSelectorsMap()[strings.TrimSpace(restriction)] {
```

Comment:
I don't see us doing any case folding anywhere, which means that this matching is case sensitive, which I think makes sense, just wanted to check that this was intentional?

Reply:
Yep this is intentionally case sensitive. Though there is no real justification for it being that way, I can't someone wanting `internal` and `INTERNAL` to mean different things.

But something doesn't feel right about making this case insensitive.